locales/deDE.lua

```diff
@@ -629,8 +629,8 @@ for key,value in pairs({
 		["25 Spieler (Heroisch)"] = "25M (H)",
 		--TODO: ["Emissary Quests"] = "Emissary",
 		--TODO: [TRACKER_HEADER_WORLD_QUESTS] = "WQ",	-- ["World Quests"] = "WQ"
-		--TODO: ["WoW Anniversary"] = "Anniversary",
-		--TODO: ["Covenant:"] = "Cov:",
+		["WoW Geburtstag"] = "Anniversary",
+		["Pakt:"] = "Cov:",
```

Comment:
I'm not sure this is correct. Just a quick explanation why we use this `ABBREVIATIONS` table. We want our source lines in ATT tooltips to be shorter when possible so instead of showing the full name (left side in this table), we abbreviate them to shorter form (right side). I should also note that it only works if the full name is the exact match of how it appears in game before any abbreviations (just find the Thing with this line in tooltip).
You've modified two lines here:
- if `WoW Geburtstag` is how it appears now, then shorter form should be `Geburtstag` I guess?
- similar with `Pakt:`, if it's really how `Covenant:` shows in German client then you probably don't even need any abbreviations, it's short enough. Can't believe German word is shorter than English! 🤣

Reply:
Hi, thank you for the thorough feedback. I believe I fixed it in my newest commit, but to be honest it's my first time touching any LUA files so let me know if it's still wrong.

Reply:
It's cool, if you have any Lua syntax errors you'll see it in-game on your load.
And thanks for changing this, I'm just gonna remove the Covenant line completely cause it doesn't do anything.